webapp/src/sidepanel.tsx

```diff
@@ -108,20 +146,33 @@ export class Sidepanel extends data.Component<SidepanelProps, SidepanelState> {
     protected handleSimPanelRef = (c: HTMLDivElement) => {
         this.simPanelRef = c;
         if (c && typeof ResizeObserver !== "undefined") {
-            const observer = new ResizeObserver(() => {
+            this.simPanelResizeObserver = new ResizeObserver(() => {
                 const scrollVisible = c.scrollHeight > c.clientHeight;
                 if (scrollVisible)
                     this.simPanelRef?.classList.remove("invisibleScrollbar");
                 else
                     this.simPanelRef?.classList.add("invisibleScrollbar");
             })
-            observer.observe(c);
+            this.simPanelResizeObserver.observe(c);
+        }
+    }
+
+    protected handleSimRef = (c: HTMLDivElement) => {
+        this.simRef = c;
+        if (c && typeof ResizeObserver !== "undefined") {
```

Comment:
I notice we do the same check above. Is this something special with ResizeObservers? I would figure the type of a ResizeObserver would always be an object, but I don't know a whole lot about them.

Reply:
I thought it was weird too... I'm not sure if there's actually an environment where they wouldn't exist, but I did the check here since it was done above, and I find the engineers of days past usually know more about this stuff than I do 😄